src/features/employees/components/EmployeeForm.tsx

```diff
@@ -0,0 +1,178 @@
+import { useState } from "react";
+import type { EmployeeFormData, EmployeeFormProps } from "../types";
+import {
+  employmentStatuses,
+  contractTypes,
+  departments,
+  roles,
+} from "@/data/lookups";
+
+export const EmployeeForm: React.FC<EmployeeFormProps> = ({
+  initialValues,
+  onSubmit,
+  onCancel,
+}) => {
+  const [form, setForm] = useState<EmployeeFormData>(initialValues);
+
+  const handleChange = (field: keyof EmployeeFormData, value: any) => {
+    setForm((prev) => ({ ...prev, [field]: value }));
+  };
+
+  const handleEmergencyChange = (field: "name" | "phone", value: string) => {
+    setForm((prev) => ({
+      ...prev,
+      emergencyContact: { ...prev.emergencyContact, [field]: value },
+    }));
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    onSubmit(form);
+  };
+
+  return (
+    <form onSubmit={handleSubmit} className="space-y-4">
+      <fieldset className="flex gap-4">
+        <label htmlFor="">
+          <input
+            type="text"
+            placeholder="First Name"
+            value={form.firstName}
+            onChange={(e) => handleChange("firstName", e.target.value)}
+            className="border rounded px-3 py-2 w-full"
+          />
+        </label>
+        <label htmlFor="">
+          <input
+            type="text"
+            placeholder="Last Name"
+            value={form.lastName}
+            onChange={(e) => handleChange("lastName", e.target.value)}
+            className="border rounded px-3 py-2 w-full"
+          />
+        </label>
+      </fieldset>
+      <fieldset className="flex gap-4">
+        <label htmlFor="">
+          <input
+            type="email"
+            placeholder="Email"
+            value={form.email}
+            onChange={(e) => handleChange("email", e.target.value)}
+            className="border rounded px-3 py-2 w-full"
+          />
+        </label>
+        <label htmlFor="">
+          <input
+            type="tel"
+            placeholder="Phone"
+            value={form.phone}
+            onChange={(e) => handleChange("phone", e.target.value)}
+            className="border rounded px-3 py-2 w-full"
+          />
+        </label>
+      </fieldset>
+
+      <select
```

Comment:
I though you already had a component for selectdropdown but it looks like you are using a it here redundantly